### PR TITLE
Deprecate version parameter in Bio.motifs.Motif's weblogo method

### LINKED ISSUE #4576: Fix motif.Motif.weblogo API version
Currently adapted to WebLogo v2.8.2 and the last version is 3.7.12!

### PR DESCRIPTION
**Pull request by michaelfm1211**
- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit``
locally, and understand that continuous integration checks will be used to
confirm the Biopython unit tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

Closes #4576

This PR implements @peterjc's suggestions from the discussion of issue #4576. The title of the PR should also be fairly self-explanatory.

(PS: on the linked issue there seemed to have been another offer to write up the deprecation, however, I couldn't find any associated PRs and it looked abandoned)

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -75,6 +75,9 @@ attributes of the ``SeqFeature`` which was done without a deprecation period.
 They are again aliases for ``.location.strand`` etc, but trigger deprecation
 warnings.
 
+The unused ``version`` parameter to the ``weblogo`` method of
+``Bio.motifs.Motif`` is also now deprecated.
+
```
Comment (peterjc) [inline NEWS.rst:80]: I think this is too minor for the NEWS file - in general things like this only go in the DEPRECATED file.

**Review by peterjc — CHANGES_REQUESTED** [review]
Remove the NEWS entry and this looks ready to me. Thank you.

**Review by mdehoon — APPROVED** [review]

File: NEWS.rst (hunk 1 above)
Comment (michaelfm1211) [reply NEWS.rst]: OK, I just pushed another commit that removes the NEWS entry.

**Review by peterjc — APPROVED** [review]

**Comment by peterjc** [comment]
Merged, thank you.

**Pull request merged**